draft-ietf-quic-transport.md

```diff
@@ -1351,6 +1355,14 @@ connection when they move networks.  This includes state that can be hard to
 recover such as outstanding requests, which might otherwise be lost with no easy
 way to retry them.
 
+An endpoint that receives packets that contain a source IP address and port that
```

Comment:
This is reasonable, but I'm not sure it's what we always want.  I think we should have a special case for PING/PONG to allow proactive path validation.  If you receive only a PING, only send the PONG down that path.  If you receive other packets, then send all traffic down the new path?

Reply:
We're definitely learning that having the ability to probe is quite useful even for connection migration. I've been working on redesigning connection migration, and our experience has shown that there needs to be more nuance around connection migration than simply switching away on the first packet. I don't want normative language here yet. I would suggest, for the sake of getting this PR in, to rephrase the MUST below to be non-normative ("should start sending", not capitalized.) We can then do an issue about what to do around connection migration separately, and figure out what should be normative.

Reply:
(What does "too woolly" mean?) We as a wg haven't decided anything about connection migration yet, so I don't think we should add any normative language until we agree on what is necessary. There's a bigger design issue question here, and I think this PR is unnecessarily rushing to resolve the design issue.

Reply:
I agree, we don't know and we should discuss.  But this PR addresses problems with use of new paths that can't otherwise be addressed.  I agree with you that this isn't a great design, it's probably even terrible, but a good design will take time.  You aren't committing to this design though.  I would prefer to see this merged, while recognizing that it is bad in certain specific ways (I don't agree that this is so bad that the protocol will break, however).  I will even open the issue that explains the shortcoming of this design if we merge this.

Reply:
I don't think this is too bad, it's just one design that I don't think is adequate. Which is why I don't think there should be normative behavior here yet. I would like to build on this PR, and add normative behavior that _we all agree_ should be normative. At the moment I don't think what you're stating as normative is useful. You're requiring that an endpoint switch immediately, which is unnecessary.

FWIW, I've opened #880.